docker/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:20.17.0
+WORKDIR /usr/app
+COPY ../package*.json ./
+RUN npm cache clean --force
+COPY ../ .
```

Comment:
Maybe add a cmd or entrypoint here? Or you expect people to manually type it in their docker run command?

Reply:
Here the idea is to just create an runtime infra for the automation app to install and start, hence the entrypoint and command is not required peter, just build the image and inside the image call `npm install` and `npm start`